scripts/run-llvm-test.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+set -eu
+
+# URL to the benchmark git repository and the commit to be used
+GIT_REPOSITORY=https://github.com/EECS-NTNU/llvm-test-suite.git
```

Comment:
I am not a big fan of having a dependency on an NTNU repository in the jlm CI where I do not have any rights. I created a clone of this repository, please add the clone: `https://github.com/phate/llvm-test-suite.git`

Reply:
You could have easily been added to the existing repo.
But I've changed the URL.

Having everything related to RVSDG and jlm under phate is maybe also not ideal, since you have 22 repositories so they kind of get swallowed. One option would be to create an organization and place everything related under it, and it looks like https://github.com/rvsdg/ is available.

Reply:
I will have a look at github organizations.